pol.json

```diff
@@ -22,5 +22,13 @@
     "decimals": 18,
     "chainId": 137,
     "logoURI": "https://assets.coingecko.com/coins/images/17638/small/Ue3Hfh8.png?1630737936"
-  }
-]
\ No newline at end of file
+  },
+    {
+    "name": "PlanetIX",
+    "address": "0xe06bd4f5aac8d0aa337d13ec88db6defc6eaeefe",
+    "symbol": "IXT",
+    "decimals": 18,
+    "chainId": 137,
+    "logoURI": "https://nomics.com/_next/image?url=https%3A%2F%2Fs3.us-east-2.amazonaws.com%2Fnomics-api%2Fstatic%2Fimages%2Fcurrencies%2FIXT2.jpg&w=48&q=75"
+  },
```

Comment:
Dangling commas are also not valid in JSON 😅  As soon as I finish my current task I'll add the GH action to validate it